eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/TechnologiePlauscherlCollector.kt

```diff
@@ -49,4 +51,14 @@ class TechnologiePlauscherlCollector : EventCollector {
 
         return events
     }
+
+    private fun prepareSyndFeed(url: String): SyndFeed {
+        HttpClients.createMinimal().use { client ->
```

Comment:
please use our Fetcher class for all http relevant stuff, as it implements a delay handling, sets the user-agent to boudicca and records http calls automatically in our collectors.boudicca.event overview

Reply:
the XmlReader with an url as param has been deprecated for quite some time, if you can create a synd feed with the Fetcher, pls tell me how to do it, otherwise this would require major refactoring of either the Fetcher or the Techplauscherl Collector

Reply:
i just converted the fetched content to an InputStream, not the most elegant solution, but it works...